### ISSUE TITLE
Updating Midevals column in google sheet through a script

### ISSUE THREAD

**Issue by rakaar** [kossiitkgp/kwoc-stats-api#8]
Background-
A CSV file is generated by the code in PR #6
A Google sheet is updated with the students and required columns through the code in PR #7 

Mid evals criteria: all students who have no commits(can be found through `stats.json`) will NOT clear the mid evals. 
After mid evals, we need the column mid_evals_cleared to be updated.
Hence, write a script that uses the CSV file(students with no commits) and updates the column `mid_evals_cleared` in the google sheet.

In other words,
students present in the CSV file(no commits CSV file) should have their mid_evals_cleared set to 0 (default value)
Other students should have `mid_evals_cleared` set to `1`
